scripts/kcl/mod1/tests/deploy-cluster-tests.sh

```
@@ -0,0 +1,437 @@
+run_testkube_api_tests() {
+    print_status "INFO" "Running Testkube tests via API..."
+    
+    # Port-forward to Testkube API
+    kubectl port-forward -n testkube svc/testkube-api-server 8088:8088 &
+    local port_forward_pid=$!
+    
+    # Wait for port-forward to be ready
+    sleep 5
+    
+    # Run test via API
+    local test_result=$(curl -s -X POST "http://localhost:8088/v1/tests/kcl-function-test/executions" \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)